src/Sylius/Bundle/CoreBundle/Fixture/Factory/UserExampleFactory.php

```diff
@@ -36,6 +36,11 @@
     private $customerFactory;
 
     /**
+     * @var RepositoryInterface
```

Comment:
It is not a part of this PR, but just a short thought about Sylius repository interfaces in general. WDYT about creating a repository interface for each class, that would at least extend a general RepositoryInterface? Even in this class constructor depends on two different repositories with the same typehint. It is super easy to change an order of them by an accident. If we would create these interfaces/markers we would received a message, that something is not properly injected much faster. 

I will probably create a separate RFC for that.